tests/helpers/pactHelper.js

```diff
@@ -208,7 +208,7 @@ const deleteResourceInteraction = (
   if (resource.includes('nonExistent')) {
     response = {
       status: 404,
-      headers: applicationXmlResponseHeaders,
+      headers: xmlResponseHeaders,
```

Comment:
Also, can we treat `application/xml` and `text/xml` as same?

Reply:
It think yes, see https://github.com/owncloud/ocis/issues/1686